aws-android-sdk-appsync-api/build.gradle

```diff
@@ -5,10 +5,10 @@ apply from: rootProject.file('gradle-mvn-push.gradle')
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.google.code.findbugs:jsr305:3.0.1' // compileOnly
-    compile 'javax.annotation:jsr250-api:1.0' // compileOnly
+    implementation 'com.google.code.findbugs:jsr305:3.0.1' // compileOnly
```

Comment:
These dependencies do not match the published POMs. https://repo.maven.apache.org/maven2/com/amazonaws/aws-android-sdk-appsync-api/2.8.1/aws-android-sdk-appsync-api-2.8.1.pom